spring-boot/admin/src/main/resources/templates/admin/fragments/api-keys/consumer/list.html

```diff
@@ -5,32 +5,11 @@
             <div class="x_panel">
                 <div class="x_content">
                     <i class="fa fa-lg fa-info-circle"/>
-                    <span id='message'>message</span>
-
+                    <span id='confirmApiKeyRemovalmessage'>message</span>
                 </div>
             </div>
         </div>
 
-        <script>
```

Comment:
Maybe removing this was the reason for the infoDialog showing up?

Reply:
If the problem does not occur any more, then stop bothering ;)

Reply:
I stopped bothering about this problem, so this can be considered done :)